bundles/com.salesforce.bazel.sdk/src/com/salesforce/bazel/sdk/model/TargetInternal.java

```diff
@@ -0,0 +1,30 @@
+/*-
+ * Copyright (c) 2024 Salesforce and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *      Salesforce - adapted from M2E, JDT or other Eclipse project
+ */
+package com.salesforce.bazel.sdk.model;
```

Comment:
Suggestion: package `com.salesforce.bazel.sdk.command.querylight` for all of these and use the suffix `Light` instead of `Internal` for these?

Actually my preference would be no suffix at all and just deal with the full qualified conflicts here. Requires less changes outside.

Reply:
Also note, adding a new package requires updating the MANIFEST.MF file export packages list.